src/client/store/ui/tablePaginated/actions/getData.ts

```diff
@@ -4,31 +4,36 @@ import axios from 'axios'
 import { TablePaginatedDataRequestParams } from 'meta/api/request/tablePaginated'
 import { AreaCode } from 'meta/area'
 import { AssessmentName, CycleName, SectionName } from 'meta/assessment'
-import { TablePaginatedOrderBy } from 'meta/tablePaginated'
+import { TablePaginatedFilterValues, TablePaginatedOrderBy } from 'meta/tablePaginated'
+import { encodeFilters } from 'meta/tablePaginated/utils'
 
 type Props = {
   assessmentName: AssessmentName
-  cycleName: CycleName
   countryIso?: AreaCode
-  sectionName?: SectionName
+  cycleName: CycleName
+  filters?: Record<string, TablePaginatedFilterValues>
+  limit: number
   orderBy?: TablePaginatedOrderBy
   page: number
   path: string
-  limit: number
+  sectionName?: SectionName
 }
```

Comment:
I would have had to do something like this because some types weren't the same:
```
type Props = Omit<TablePaginatedDataRequestParams, 'limit' | 'filters' | 'offset' | 'orderBy' | 'orderByDirection'> & {
  filters?: Record<string, TablePaginatedFilterValues>
  limit: number
  orderBy?: TablePaginatedOrderBy
  page: number
  path: string
}
```
So I better exposed `TablePaginatedBaseParams` and used it here and in `getCount`. Let me know what you think.